CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+### 0.12.9
+*Features*
+- Added a helper function to allow retrying blocks of code
```

Comment:
Maybe I'm wrong about this, but I'd say the changelog should be for users trying to decide which version to pick (so internal code details are not worth mentioning)